src/app/modules/recipe-list/components/recipe-sort/recipe-sort.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <mat-accordion class="container__accordion">
+    <mat-expansion-panel>
+      <mat-expansion-panel-header class="small-font">
+        <mat-panel-title>
+          Sortuj
+        </mat-panel-title>
+      </mat-expansion-panel-header>
+      <div class="sorting-options">
+        <button *ngFor="let o of sortNames" mat-button class="single-option" (click)="onSortOption(o)">{{o}}</button>
```

Comment:
please always use full name
```suggestion
        <button *ngFor="let sortName of sortNames" mat-button class="single-option" (click)="onSortOption(sortName)">{{sortName}}</button>
```